SWMessages/Source/SWMessageView.swift

```diff
@@ -218,6 +231,21 @@ public class SWMessageView :UIView , UIGestureRecognizerDelegate {
         fatalError("init(coder:) has not been implemented")
     }
     
+    public override func drawRect(rect: CGRect) {
+        super.drawRect(rect)
+        let options = SWMessageView.styleForMessageType(notificationType)
```

Comment:
I think we have to handle style  passed through constructor here? 

maybe like self.style in constructor, and use it here?